release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1573,6 +1576,35 @@ sourceStrategy:
 
 * With this update, you can now set the security context for the registry pod by including the `securityContext` configuration field in the pod specification. This will apply the security context for all containers in the pod. The `securityContext` field also defines the pod's privileges. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2091864[*BZ#2091864*])
 
+[discrete]
+[id="ocp-4-12-file-integrity-operator-bug-fixes"]
+==== File Integrity Operator
+
+* Previously, the File Integrity Operator deployed templates using the `openshift-file-integrity` namespace in the permissions for the Operator. When the Operator attempted to create objects in the namespace, to would fail due to permission issues. With this release, the deployment resources used by OLM are updated to use the correct namespace, fixing the permission issues so that users can install and use the operator in a non-default namespaces. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2104897[*BZ#2104897*])
```

Comment:
```suggestion
* Previously, the File Integrity Operator deployed templates using the `openshift-file-integrity` namespace in the permissions for the Operator. When the Operator attempted to create objects in the namespace, it would fail due to permission issues. With this release, the deployment resources used by OLM are updated to use the correct namespace, fixing the permission issues so that users can install and use the operator in non-default namespaces. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2104897[*BZ#2104897*])
```